calculation/gmhazard_calc/gmhazard_calc/verification/GMM/constants.py

```diff
@@ -51,73 +37,66 @@
 
 # Just for psha_psa_plots.py
 PSA_IM_NAME = "pSA"
+PGA_IM_NAME = "PGA"
 
-# Based on empirical_db/empirical_model_configs/20p11.yaml
+# Based on db_creation/empirical_db/empirical_model_configs/21p10.yaml
 ss_spectral_models = [
-    "ZA_06",
-    "A_18",
-    "BCH_16",
-    "P_20_SS",
-    "HA_20_SS",
-    "PH_20_SS",
-    "CH_20_SS",
-    "AG_20_SS",
-    "AG_20_SS_NZ",
-    "K_20_SS",
-    "K_20_SS_NZ",
+    # "ZA_06",
+    # "BCH_16",
+    # "A_18",
```

Comment:
I told this Joel in person but leaving a comment so that we are all on the same page.
![image](https://user-images.githubusercontent.com/7849113/184735538-34ad79d8-1342-46d6-add6-229f4395e775.png)

Without those specially requested plots, we used to use all that models, ZA_06, BCH_16 and A_18.
However, those with specially requested plots didn't require those models. Hence, I temporarily disabled but in the end, Robin will be the one to modify any models to plot with.